### ISSUE TITLE
Volcano job natively support Ray framework

### ISSUE THREAD

**Issue by Monokaix** [volcano-sh/volcano#4182]
### What is the problem you're trying to solve

Ray is a popular AI framework that has been widely used, and ray operator has already supported Volcano as a batch scheduler, see：https://github.com/ray-project/kuberay
On the other hand, many users still use volcano job to run their job instead of new ray API，so it's reasonable to support ray natively on volcano job.

### Describe the solution you'd like

Currently volcano supports distributed AI and HPC framework like pytorch, tensorflow,  mpi.
Just like what vcjobs currently do, and a new plugin named `ray` in pkg/controllers/job/plugins/distributed-framework, so users can submit a vcjob and actually run a ray job.

**Comment by Monokaix** [comment]
> /assign

Sorry, this issue has been assigned to @MondayCha and you can find other issues that need to be resolved: )

**Comment by blazethunderstorm** [comment]
@JesseStutler @Monokaix can I work on this pls

**Comment by blazethunderstorm** [comment]
@JesseStutler I have solved issue can I raise pr

**Comment by JesseStutler** [comment]
@de6p Have you done it ?

**Comment by de6p** [comment]
https://github.com/volcano-sh/volcano/pull/4193

**Comment by Wonki4** [comment]
@de6p 
Excuse me, Could you consider using my code for ray plugin?
For #4581 , the code, architecture, docs, user-guide and tests(UT & E2E) have been prepared.
